README.md

```diff
@@ -4,6 +4,26 @@
 
 This is a [Next.js](https://nextjs.org/) v12 project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app). It is a reference on how to integrate commonly used features within Netlify for Next.js. 
 
+## Table of Contents:
+
+- [Getting Started](#getting-started)
+- [Deploy to Netlify](#deploy-to-netlify)
+  - [Deploy using the Netlify CLI](#deploy-using-the-netlify-cli)
+  - [Running Locally](#running-locally)
+- [Forms](#forms)
+  - [Adding a Custom Submission Page](#adding-a-custom-submission-page)
+  - [Blocking Bot Spam with a Honeypot Field](#blocking-bot-spam-with-a-honeypot-field)
+  - [Forms Resources](#forms-resources)
+- [Netlify Functions](#netlify-functions)
+  - [Functions Resources](#functions-resources)
+- [Redirects](#redirects)
+  - [Redirect Resources](#redirect-resources)
+- [Next.js Toolbox Template Walkthrough](#nextjs-toolbox-template-walkthrough)
```

Comment:
I'm open to renaming the heading to be just like what you have in the TOC but right now the link to the template walkthrough doesn't work because it is missing `video` in it.
```suggestion
- [Next.js Toolbox Template Walkthrough](#nextjs-toolbox-template-video-walkthrough)
```